diff --git a/node_modules/redux/dist/redux.js b/node_modules/redux/dist/redux.js
index 600b255..5dafa9a 100644
--- a/node_modules/redux/dist/redux.js
+++ b/node_modules/redux/dist/redux.js
@@ -752,7 +752,7 @@ function applyMiddleware() {
 function isCrushed() {}
 
 if (typeof isCrushed.name === 'string' && isCrushed.name !== 'isCrushed') {
-  warning('You are currently using minified code outside of NODE_ENV === "production". ' + 'This means that you are running a slower development build of Redux. ' + 'You can use loose-envify (https://github.com/zertosh/loose-envify) for browserify ' + 'or setting mode to production in webpack (https://webpack.js.org/concepts/mode/) ' + 'to ensure you have the correct code for your production build.');
+  console.log('You are currently using minified code outside of NODE_ENV === "production". ' + 'This means that you are running a slower development build of Redux. ' + 'You can use loose-envify (https://github.com/zertosh/loose-envify) for browserify ' + 'or setting mode to production in webpack (https://webpack.js.org/concepts/mode/) ' + 'to ensure you have the correct code for your production build.');
 }
 
 exports.__DO_NOT_USE__ActionTypes = ActionTypes;
diff --git a/node_modules/redux/es/redux.js b/node_modules/redux/es/redux.js
index 1c0c9ab..27e2341 100644
--- a/node_modules/redux/es/redux.js
+++ b/node_modules/redux/es/redux.js
@@ -710,7 +710,7 @@ function applyMiddleware() {
 function isCrushed() {}
 
 if (process.env.NODE_ENV !== 'production' && typeof isCrushed.name === 'string' && isCrushed.name !== 'isCrushed') {
-  warning('You are currently using minified code outside of NODE_ENV === "production". ' + 'This means that you are running a slower development build of Redux. ' + 'You can use loose-envify (https://github.com/zertosh/loose-envify) for browserify ' + 'or setting mode to production in webpack (https://webpack.js.org/concepts/mode/) ' + 'to ensure you have the correct code for your production build.');
+  console.log('You are currently using minified code outside of NODE_ENV === "production". ' + 'This means that you are running a slower development build of Redux. ' + 'You can use loose-envify (https://github.com/zertosh/loose-envify) for browserify ' + 'or setting mode to production in webpack (https://webpack.js.org/concepts/mode/) ' + 'to ensure you have the correct code for your production build.');
 }
 
 export { ActionTypes as __DO_NOT_USE__ActionTypes, applyMiddleware, bindActionCreators, combineReducers, compose, createStore, legacy_createStore };
diff --git a/node_modules/redux/lib/redux.js b/node_modules/redux/lib/redux.js
index 9218f82..2449d38 100644
--- a/node_modules/redux/lib/redux.js
+++ b/node_modules/redux/lib/redux.js
@@ -718,7 +718,7 @@ function applyMiddleware() {
 function isCrushed() {}
 
 if (process.env.NODE_ENV !== 'production' && typeof isCrushed.name === 'string' && isCrushed.name !== 'isCrushed') {
-  warning('You are currently using minified code outside of NODE_ENV === "production". ' + 'This means that you are running a slower development build of Redux. ' + 'You can use loose-envify (https://github.com/zertosh/loose-envify) for browserify ' + 'or setting mode to production in webpack (https://webpack.js.org/concepts/mode/) ' + 'to ensure you have the correct code for your production build.');
+  console.log('You are currently using minified code outside of NODE_ENV === "production". ' + 'This means that you are running a slower development build of Redux. ' + 'You can use loose-envify (https://github.com/zertosh/loose-envify) for browserify ' + 'or setting mode to production in webpack (https://webpack.js.org/concepts/mode/) ' + 'to ensure you have the correct code for your production build.');
 }
 
 exports.__DO_NOT_USE__ActionTypes = ActionTypes;
diff --git a/node_modules/redux/src/index.js b/node_modules/redux/src/index.js
index 141a84e..4a1acb9 100644
--- a/node_modules/redux/src/index.js
+++ b/node_modules/redux/src/index.js
@@ -17,7 +17,7 @@ if (
   typeof isCrushed.name === 'string' &&
   isCrushed.name !== 'isCrushed'
 ) {
-  warning(
+  console.log(
     'You are currently using minified code outside of NODE_ENV === "production". ' +
       'This means that you are running a slower development build of Redux. ' +
       'You can use loose-envify (https://github.com/zertosh/loose-envify) for browserify ' +
